sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/Spark33XShims.scala

```diff
@@ -56,4 +57,24 @@ trait Spark33XShims extends Spark33XFileOptionsShims {
     new ParquetFilters(schema, pushDownDate, pushDownTimestamp, pushDownDecimal, pushDownStartWith,
       pushDownInFilterThreshold, caseSensitive, datetimeRebaseMode)
   }
+
+  override def getExprs: Map[Class[_ <: Expression], ExprRule[_ <: Expression]] = {
+    (super.getExprs.values ++ Seq(
+      GpuOverrides.expr[BitLength](
```

Comment:
BitLength has been in Spark since 2.3.0. Why are we only putting it into the 3.3 shim?

Reply:
Because they were not available for Spark users through any explicit way before Spark 330.

Reply:
https://github.com/apache/spark/blob/3fdfce3120f307147244e5eaf46d61419a723d50/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L347 and https://github.com/apache/spark/blob/3fdfce3120f307147244e5eaf46d61419a723d50/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L329

They are in the SQL function registry for v3.0.0. They appear to be available for all versions of Spark that we support. I don't think you would even need to change the test because you are calling them from the SQL functions.  In 3.3 they scala APIs were added to make it simpler to call them, but they have been available for a long time.

Reply:
I moved these bindings into the common section.